static/js/apps/eval_retrieval_generation/sxs/app.tsx

```diff
@@ -84,41 +84,21 @@ export function App(props: AppPropType): JSX.Element {
       )}
 
       {user && <p>Signed in as {user.email}</p>}
-      {user && !docInfo && <p>Loading query...</p>}
-      {docInfo && (
-        <>
+      {user && !docInfos && <p>Loading query...</p>}
+      {docInfos && (
+        <AppContext.Provider
```

Comment:
this app context doesn't seem necessary since none of the child components are actually accessing this context?

Reply:
Hm yeah I thought it might be necessary in the future but now that I've started working on the feedback component I agree it can be removed.